infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/Annotations.java

```diff
@@ -37,6 +37,8 @@
   public static final String SERVER_ATTR_ANNOTATION_FMT =
       ANNOTATION_PREFIX + "server.%s.attributes";
 
+  public static final String EXPOSED_MACHINE_ANNOTATION = ANNOTATION_PREFIX + "exposed_machine";
```

Comment:
Can we avoid adding a new term `exposed machine` and use something like `machine_name`